tests/test_acceleration_pool.py

```diff
@@ -71,3 +76,16 @@ def test_acceleration_pool_delete_bad_status(self):
             self.assertRaises(exceptions.PowerFlexFailDeleting,
                               self.client.acceleration_pool.delete,
                               self.fake_ap_id)
+
+    def test_acceleration_pool_query_selected_statistics(self):
```

Comment:
@ronhakim Can you please add the assertion for the unit test.

Please update the same for the other objects as well.

Reply:
@sachin-apa, asserts added as suggested. Please review again.